### ISSUE TITLE
Calibration info for LabelManager PnP, 12mm/9mm tape sizes

### ISSUE THREAD

**Issue by maresb** [labelle-org/labelle#60]
Here are three labels. The first one is a sample pattern with height 512 on 12mm tape. The pattern is truncated at row 64, so I conclude that the print head is 64 pixels high.

I measure the sample pattern to be about 8¾mm high.

![image](https://github.com/labelle-org/labelle/assets/15216687/ccf72712-1670-4b3d-bf45-79def07caf61)

The next two show the 64 pixel sample pattern on 12mm and 9mm tapes.

* 12mm
  * Top margin 1½mm
  * Bottom margin 1½mm
 
* 9mm
  * Top margin: 64th row is partially visible at the tape's edge
  * Bottom margin: 1st row is partially visible at the tape's edge

Is this all the info we need? @FaBjE, @tomek-szczesny, @tomers?

**Comment by tomek-szczesny** [comment]
Yes, this is all the information we could possibly ask for. Looks way better than the original one. :) 

h=512 obviously doesn't fit on the print, I wonder if there is anything we could do to preserve this information regardless of the tape size. How about repeating "h=xxx" across the full height of the print?

**Comment by maresb** [comment]
I could do that, but we're only going to do this a dozen or so times, and I don't want to go through the effort to recreate the photos yet again unless there's a really good reason. I'd say that simply not having h=xyz on the pattern is enough evidence that the pattern height is sufficiently large, especially since we don't actually care what h is.

**Comment by varmfskii** [comment]
On a 19mm tape I get the bottom approximately 24 pixels cut off and anything above about 224 cut off on the top.
DYMO LabelManager Plug N Play Label Maker

**Comment by maresb** [comment]
@varmfskii, wow it looks like your printer is barely able to print. I've never seen anything nearly this extreme before. Could you check the following?

https://github.com/labelle-org/labelle/blob/main/doc/margin-calibration-howto.md#checking-the-cleanliness-of-the-print-head
